source/WindowScroller/WindowScroller.js

```diff
@@ -138,22 +135,22 @@ export default class WindowScroller extends React.PureComponent<Props, State> {
     this._isMounted = true;
   }
 
-  componentWillReceiveProps(nextProps: Props) {
-    const scrollElement = this.props.scrollElement;
-    const nextScrollElement = nextProps.scrollElement;
+  componentDidUpdate(prevProps: Props, prevState: State) {
```

Comment:
do we need to include `prevState` if we're not using it?

Reply:
I guess we don't _need_ to but it's part of the lifecycles signature so it's just habit to add it. 😁